### PR TITLE
feat: investor transaction list CSV

### PR DESCRIPTION
**Pull request by AStox**
### Description

This pull request adds a button to download the new investor transaction list CSV

### Approvals

- [x] Dev
- [x] Designer
- [ ] Product

### Screenshots
image

### Impact

This pull request impacts the profile page only.

### CODE REVIEW

File: tinlake-ui/containers/Portfolio/index.tsx [hunk 1]
```diff
@@ -199,7 +209,13 @@ const Portfolio: React.FC<Props> = (props: Props) => {
 
           <Box margin={{ top: 'medium', left: 'auto', right: 'auto' }}>
             <Link href="/">
-              <Button label="Explore further investment opportunities" secondary size="small" />
```
Comment (armandomistral) [inline tinlake-ui/containers/Portfolio/index.tsx:202]: @AStox, let's stick to using the button component here. But let's use type tertiary/text for it. Here's the hierarchy of button styles:
image

File: tinlake-ui/containers/Portfolio/index.tsx [hunk 2]
```diff
@@ -122,6 +124,14 @@ const Portfolio: React.FC<Props> = (props: Props) => {
               />
             </Box>
           )}
+          {address && hasBalance && (
+            <Box direction="row" align="center">
+              <FormNextLink style={{ transform: 'rotate(90deg)', marginRight: '5px' }} />
+              <Heading level={4} onClick={() => investorTransactions(address)} style={{ cursor: 'pointer' }}>
+                Download transactions
```
Comment (armandomistral) [inline tinlake-ui/containers/Portfolio/index.tsx:131]: @AStox, let's use the button component of type "text/tertiary" for this action here (see other comment)…

**Review by armandomistral — CHANGES_REQUESTED** [review]
Use button component of type tertiary/text instead of box component for actions `Download transactions`, and `Browse pools`

File: tinlake-ui/containers/Portfolio/index.tsx (hunk 1 above)
Comment (onnovisser) [reply tinlake-ui/containers/Portfolio/index.tsx:202]: @AStox The Axis Button has a "plain" variant
```
<Button
  plain
  icon={<FormNextLink style={{ transform: 'rotate(90deg)' }} />}
  label="Explore further investment opportunities"
/>
```

File: tinlake-ui/containers/Portfolio/index.tsx (hunk 2 above)
Comment (AStox) [reply tinlake-ui/containers/Portfolio/index.tsx]: I dont actually see this button type anywhere. But the one Onno suggested seems to work well:
`<Button
  plain
  icon={<FormNextLink />}
  label="Explore further investment opportunities"
/>`

Comment (armandomistral) [reply tinlake-ui/containers/Portfolio/index.tsx]: Perfect, "plain" is the one! I was not aware it's called this way in code. Changed the name to "plain" the in design system.

**Review by jpangelle — APPROVED** [review]
👍

**Review by onnovisser — APPROVED** [review]

File: tinlake-ui/containers/Portfolio/index.tsx [hunk 3]
```diff
@@ -122,6 +124,15 @@ const Portfolio: React.FC<Props> = (props: Props) => {
               />
             </Box>
           )}
+          {address && hasBalance && (
+            <Button
+              label="Download transactions"
+              plain
+              onClick={() => investorTransactions(address)}
+              icon={<FormNextLink style={{ transform: 'rotate(90deg)' }} />}
```
Comment (armandomistral) [inline tinlake-ui/containers/Portfolio/index.tsx:132]: @AStox, I've added an "arrow-down" icon to brand assets here: https://www.figma.com/file/Xbfgir0ZyOjgteEftT8sxY/Brand-assets?node-id=3421%3A541

**Review by armandomistral — APPROVED** [review]

**Review by armandomistral — CHANGES_REQUESTED** [review]
@AStox, please make the icon also turn blue when hovering over the button. This should be the standard behavior of the component. Currently the icon stays black when hovering, see https://www.loom.com/share/e6f6c9883fff47fb9d7a727dec5fe16b

**Pull request closed without merging**